uml/sequence_diagrams/Power.txt

```diff
@@ -0,0 +1,51 @@
+@startuml
+participant Acceletometer
+participant PowerMonitor
+participant Battery
+participant Brightness
+participant Storage
+participant VibrationMotor
+
+loop event
+Alt event = timerBattery
+PowerMonitor --> Battery: AskBattery
+Activate Battery
+Battery --> PowerMonitor: Battery
+Deactivate Battery
+alt (Battery == BatteryError OR Battery<=10%) AND LowPower == OFF

```

Comment:
Reminder to account for 2% SoC threshold shown in corresponding state diagram

Reply:
fixed